lib/normalizeOptions.js

```diff
@@ -65,7 +65,7 @@ function normalizeOptions(loaderContext, content, webpackImporter) {
     }
 
     // Allow passing custom importers to `node-sass`. Accepts `Function` or an array of `Function`s.
-    options.importer = options.importer ? proxyCustomImporters(options.importer, resourcePath) : [];
+    options.importer = options.importer ? proxyCustomImporters(options.importer, resourcePath, loaderContext) : [];
```

Comment:
In case this is lands `resourcePath` && `loaderContext` should be merged here, since `resourcePath` is already on the `loaderContext`